config.h.meson

```diff
@@ -0,0 +1,98 @@
+/* config.h.in.  Generated from configure.ac by autoheader.  */
```

Comment:
This is the simplest way of getting a build going is to do this. Meson does not actually need a `config.h.in`, if you don't have one it will just serialise out values set in the configuration object. Which one to use depends on how the project maintainers want to set things up.